packages/cli/src/commands/members/add.ts

```diff
@@ -0,0 +1,33 @@
+import {Command, flags} from '@heroku-cli/command'
+import {Args} from '@oclif/core'
+import * as Heroku from '@heroku-cli/schema'
+const {RoleCompletion} = require('@heroku-cli/command/lib/completions')
+import {addMemberToTeam, inviteMemberToTeam} from '../../lib/members/util'
+export default class MembersAdd extends Command {
+    static topic = 'members';
+    static description = 'adds a user to a team';
+    static flags = {
+      role: flags.string({char: 'r', required: true, description: 'member role (admin, collaborator, member, owner)', completion: RoleCompletion}),
+      team: flags.team({required: true}),
+    };
+
+    static args = {
+      email: Args.string({required: true}),
+    };
+
+    public async run(): Promise<void> {
+      const {flags, args} = await this.parse(MembersAdd)
+      const {team, role} = flags
+      const {body: teamInfo} = await this.heroku.get<Heroku.Team>(`/teams/${team}`)
+      const email = args.email
+      const {body: groupFeatures} = await this.heroku.get<Heroku.TeamFeature[]>(`/teams/${team}/features`)
+
+      if (teamInfo.type === 'team' && groupFeatures.find(feature => {
```

Comment:
eslint is complaining about `find` on my machine and wants to replace it with `some`. Are you seeing this too?

Reply:
Yeah, I just didn't pay attention on this one for some reason. Fixed. Thank you!